+eui/FieldPanel.m

```diff
@@ -0,0 +1,229 @@
+classdef FieldPanel < handle
```

Comment:
why call this 'FieldPanel' instead of something like 'GlobalPanel'?

Reply:
This is what Chris originally called it.  It's a panel that generates UI fields.  I suppose it's more functionally clear than 'global' which is a less concrete name